app/Http/Controllers/UserSettingsController.php

```diff
@@ -12,15 +12,15 @@ public function index()
         return view('auth.settings');
     }
 
-    public function store_password(Request $request){
+    public function store_password(Request $request) {
```

Comment:
__[code_sniffer]__
* Method name "UserSettingsController::store_password" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->